### PR TITLE
allow direct access grants

### PR DESCRIPTION
**Pull request by philbrookes**
## Motivation
oc login on CLI was not functioning

## Verification
`oc login https://master.pbrookes-7081.openshiftworkshop.com`

### CODE REVIEW

File: evals/roles/rhsso/templates/keycloak-realm.json.j2 [hunk 1]
```diff
@@ -81,7 +81,8 @@
                     "{{ rhsso_redirect_uri }}/*",
                     "http://localhost:3006*"
                 ],
-                "standardFlowEnabled": true
+                "standardFlowEnabled": true,
+                "directAccessGrantsEnabled": true,
```
Comment (pmccarthy) [inline evals/roles/rhsso/templates/keycloak-realm.json.j2]: Should this have a trailing comma at the end?

Comment (philbrookes) [reply evals/roles/rhsso/templates/keycloak-realm.json.j2]: no, it's gone now.

**Comment by camilamacedo86** [comment]
@philbrookes could you rebase this with the master? I will test it now.

**Comment by camilamacedo86** [comment]
Starting to test it now :-)

**Pull request merged**